components/buttons/WideFoundationButton.tsx

```diff
@@ -11,10 +11,17 @@ export default function WideFoundationButton({
   className?: string
 }) {
   return (
-    <Button
-      className={`bg-foundation text-white rounded-full ${className || ''}`}
+    <Link
```

Comment:
Buttons like Message Seller only work if you click on the button text at the moment!  I thought there was something wrong with their click handlers/links but then I realised the issue is only the text was being turned into a link

(I might not have got the class/css stuff right here)